packages/parameters/src/types/BaseProvider.ts

```diff
@@ -0,0 +1,34 @@
+type TransformOptions = 'auto' | 'binary' | 'json';
+
+interface GetOptionsInterface {
+  maxAge?: number
+  forceFetch?: boolean
+  sdkOptions?: unknown
+  transform?: TransformOptions
+}
+
+interface GetMultipleOptionsInterface {
+  maxAge?: number
+  forceFetch?: boolean
+  sdkOptions?: unknown
+  transform?: string
+  throwOnTransformError?: boolean
+}
```

Comment:
Just to reduce boilerplate:
```
interface GetMultipleOptionsInterface extends GetOptionsInterface {
    throwOnTransformError?: boolean
}
```

Reply:
Good point, please feel free to address this in your PR for `AppConfigprovider`